docs/database.md

```diff
@@ -21,8 +21,10 @@ The Database is split into different key spaces based on prefixing that can be u
 
 | Name                  | Prefix | Key                              | Value                           |
 |-----------------------|--------|----------------------------------|---------------------------------|
-| Pending Transactions  | `0x01`   | Transaction ID (sha256, 32bytes) | Transaction                     |
-| Accepted Transactions | `0x02`   | Transaction ID (sha256, 32bytes) | Confirmation epoch, Transaction |
+| Pending Transactions  | `0x01` | Transaction ID (sha256, 32bytes) | Transaction                     |
+| Accepted Transactions | `0x02` | Transaction ID (sha256, 32bytes) | Confirmation epoch, Transaction |
+| Drop Peer             | `0x03` | Peer ID (u16)                    |                                 |
+| Lightning Gateways    | `0x04` | Node Pubkey (PublicKey)          | LightningGateway                |
```

Comment:
If we were to move the API endpoints to the LN module the prefix also had to change.

Reply:
Done.